catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/operations/AirflowPipelineResourceTest.java

```diff
@@ -547,7 +547,7 @@ private void validateGeneratedAirflowPipelineConfig(AirflowPipeline airflowPipel
     DatabaseService databaseService = helper(airflowPipeline).findEntity("service", DATABASE_SERVICE);
     DatabaseConnection databaseConnection = databaseService.getDatabaseConnection();
     DatabaseServiceMetadataPipeline metadataPipeline =
-        JsonUtils.convertValue(airflowPipeline.getPipelineConfig(), DatabaseServiceMetadataPipeline.class);
+        JsonUtils.convertValue(airflowPipeline.getPipelineConfig().getConfig(), DatabaseServiceMetadataPipeline.class);
```

Comment:
Needed to add this when casting to `DatabaseServiceMetadataPipeline`, as the actual properties where nested in a `config` object that was now unnexpected